services/src/unit-test/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionServiceTest.java

```diff
@@ -762,12 +826,37 @@ private void setUpIndividualRequest(final IIndividualRequest request) {
         bulkSubmission.setCustomerReference("TEST_CUST_REF");
         bulkSubmission.setId(1L);
         bulkSubmission.setNumberOfRequest(1);
-        final List<IIndividualRequest> requests = new ArrayList<IIndividualRequest>();
+        final List<IIndividualRequest> requests = new ArrayList<>();
         requests.add(request);
 
         bulkSubmission.setIndividualRequests(requests);
 
         request.setBulkSubmission(bulkSubmission);
     }
 
+
+    @Test
+    void testSetIndividualRequestDaoConsumerWriter(){
+        IIndividualRequestDao individualRequestDaoMock = mock(IndividualRequestDao.class);
+        IConsumerGateway consumerGatewayMock = mock(ConsumerGateway.class);
+        IMessageWriter mockIMessageWriter = mock(MessageWriter.class);
+
+        targetAppSubmissionService.setMessageWriter(mockIMessageWriter);
+        targetAppSubmissionService.setRequestConsumer(consumerGatewayMock);
+        targetAppSubmissionService.setIndividualRequestDao(individualRequestDaoMock);
+
+        assertNotNull(targetAppSubmissionService.getIndividualRequestDao(),"Object should have been populated");
```

Comment:
Test needs to check that message writer and consumer gateway have been set as well as individual request DAO.

It'll be better to use assertEquals() to do these checks rather than assertNotNull().  The fields are set by the constructor in setUp(), so will already have a value.  Just checking whether they are null or not won't prove that the set methods worked as it could be just finding the values set by the constructor.  If you use assertEquals() you can check that they've been set to the objects created here which will prove that the set methods work.

Reply:
used the AbstractSdtUnitTest class to check this.